x-pack/plugins/security_solution/public/flyout/document_details/left/components/correlations_details.tsx

```diff
@@ -21,7 +21,7 @@ import { useShowRelatedAlertsBySession } from '../../shared/hooks/use_show_relat
 import { RelatedAlertsByAncestry } from './related_alerts_by_ancestry';
 import { SuppressedAlerts } from './suppressed_alerts';
 
-export const CORRELATIONS_TAB_ID = 'correlations-details';
+export const CORRELATIONS_TAB_ID = 'correlations';
```

Comment:
any reason for changing this constant and all the other tab ones?

Reply:
cleaning up the tab names for dashboard
![image](https://github.com/elastic/kibana/assets/18648970/ed70425c-51b9-42da-ba3a-3bb71aacf732)